app/views/studies/initialize_study.html.erb

```diff
@@ -417,6 +417,7 @@
   <div id="upload-wizard-target">
   </div>
   <script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
+    window.SCP.readOnlyToken = '<%= get_read_access_token(@study, current_user) %>'
```

Comment:
Could you add a code comment on why `readOnlyToken` is needed in the upload wizard?

With this PR context in mind, I surmise it's needed for reference images (they're read directly from the bucket, right?), but I definitely won't recall that a few days from now.  Also, it's not totally clear to me now why this would be needed when we're _writing_ the image files, as my understanding is `readOnlyToken` is only needed to read them.